.github/workflows/pr_jira_check.yml

```diff
@@ -0,0 +1,24 @@
+name: PR checker and updater for Jira missing stuff
```

Comment:
I think we should change this to `Jira ticket check`

Reply:
Updated!